README.md

```diff
@@ -16,17 +16,22 @@ Once you have implemented the `sa` program (and tested it to the best of your ab
 # Report
 
 ## Algorithm
-
+Started with SAIS, abandoned it as i didn't get the slides (and no good youtube videos exist for the recursive part). Did the naive instead. 
+Did the match algorithm using an LCP array, constructed naively.
 *Which algorithm did you use to construct the suffix array. What is its running time?*
 
 ## Insights you may have had while implementing the algorithm
 
+
 ## Problems encountered if any
+Wasn't able to implement SAIS due to lack of understanding of the recursive part (it did go well up to that point though).
 
 ## Validation
-
+Ran on several types of strings and patterns, including trusty missisippi.
 *How did you validate that the construct and the search algorithm works?*
 
 ## Running time
-
+![](match_dna.png)
+![](SA_DNA.png)
```

Comment:
I have no idea what those figures are showing. There are too few points for me even to guess.